jbot/src/main/java/me/ramswaroop/jbot/core/slack/Bot.java

```diff
@@ -151,6 +151,13 @@ public final void reply(WebSocketSession session, Event event, Message reply) {
             if (reply.getChannel() == null && event.getChannelId() != null) {
                 reply.setChannel(event.getChannelId());
             }
+            if (reply.getThreadTs() == null) {
+                if (event.getThreadTs() != null) {
+                    reply.setThreadTs(event.getThreadTs());
+                } else if (event.getTs() != null) {
```

Comment:
Wouldn't this make every reply a threaded reply?

Reply:
Yes. I'll make it an additional param in the method so it stays flexible.